### PR TITLE
added index views

### PR DESCRIPTION
**Pull request by michaelyatco**

### CODE REVIEW

File: app/views/pages/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<h1>Single Mike Menu</h1>
+<ul>
+  <li>String cheese</li>
+  <li>Ritz Crackers</li>
+  <li>Tears for seasoning</li>
```
Comment (samanthamorco) [inline app/views/pages/index.html.erb]: This is not a cheese. This should be removed.

Comment (samanthamorco) [reply app/views/pages/index.html.erb]: (Just line 5)

Comment (michaelyatco) [reply app/views/pages/index.html.erb]: Updated

**Comment by samanthamorco** [comment]
+1

**Pull request merged**